Vagrantfile

```diff
@@ -7,12 +7,12 @@ Vagrant::Config.run do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "centos-6.3-64bit-puppet-vbox.4.1.20-2"
+  config.vm.box = 'CentOS-6.4-x86_64-v20131103.box'
```

Comment:
Why are we using centos as the default?  Given the vast bulk of our machines are debian/ubuntu...